### DISCUSSION TITLE
MOVE_IN and MOVE_OUT events do not work on ZeppOS 3

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#294] [Developer Feedback]
To be able to move the widget on the screen I create events for it: CLICK_DOWN, CLICK_UP, MOVE_OUT, MOVE_IN, MOVE.
Events MOVE_IN and MOVE_OUT do not work. The other events work correctly. I create events for the STROKE_RECT widget. Events don't work in the simulator and on the watch.

**Comment by MyoungXUE** [comment]
Sorry, this problem has been confirmed for a long time. Currently, only the Button widget implements MOVE_IN and MOVE_OUT events. I will supplement the widget's support for gestures to the document.

**Comment by SashaCX75** [comment]
Thanks for the reply. I will try to use the button to track the movement.
